src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    public static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
```

Comment:
The logger should be private and not public. This encapsulates the logger within the class and prevents it from being accessed externally.